packages/custom-url/src/Infrastructure/Symfony/Serializer/CustomUrlNormalizer.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\CustomUrl\Infrastructure\Symfony\Serializer;
+
+use Sulu\CustomUrl\Domain\Model\CustomUrlInterface;
+use Symfony\Component\Serializer\Normalizer\AbstractNormalizer;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+
+final class CustomUrlNormalizer implements NormalizerInterface
+{
+    public function __construct(
+        private ObjectNormalizer $objectNormalizer
+    ) {
+    }
+
+    /**
+     * @return array<mixed>
+     */
+    public function normalize(
+        mixed $data,
+        ?string $format = null,
+        array $context = [],
+    ): array {
+        $context[AbstractNormalizer::IGNORED_ATTRIBUTES] ??= [];
+        $context[AbstractNormalizer::IGNORED_ATTRIBUTES][] = 'created';
```

Comment:
@alexander-schranz sould we move this to the auditable thing i build in this PR? https://github.com/sulu/sulu/pull/7972